### ISSUE TITLE
Pulmonary blood flow redistribution

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6709]
**Preferred term label:**
Pulmonary blood flow redistribution

**Synonyms**

**Definition (free text, please give PubMed ID)**
Pulmonary blood flow redistribution refers to any departure from the normal distribution of blood flow in the lungs that is caused by an increase in pulmonary vascular resistance elsewhere in the pulmonary vascular bed. Pulmonary blood flow redistribution is indicated by a decrease in the size and/or number of visible pulmonary vessels in one or more lung regions, with a corresponding increase in number and/or size of pulmonary vessels in other parts of the lung. 

PMID:18195376

Comment
Upper lobe blood diversion in patients with mitral valve disease is the archetypal example of redistribution. See Figure 59 of Hansell DM, et al., Fleischner Society: glossary of terms for thoracic imaging. Radiology. 2008 Mar;246(3):697-722.

Parent term (use hpo.jax.org/app)
Abnormal pulmonary thoracic imaging finding HP:0031983

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
